packages/embed/src/utils/load-options-from-attributes.spec.ts

```diff
@@ -82,6 +82,12 @@ describe('load-options-from-attributes', () => {
     })
   })
 
+  describe('to array', () => {
```

Comment:
Please also include tests for malformed strings, eg if it starts or ends with coma (`,`), if it contains white spaces around comas or inside field name.